### PR TITLE
PXB-3106 Document transition ZSTD to GA (8.0.34-29)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
renamed:    docs/archive/enabling_tcp.md -> archive/enabling_tcp.md
	renamed:    docs/archive/incremental_backups.md -> archive/incremental_backups.md
	renamed:    docs/archive/recipes_xbk_compressed.md -> archive/recipes_xbk_compressed.md
	renamed:    docs/archive/recipes_xbk_full.md -> archive/recipes_xbk_full.md
	renamed:    docs/archive/recipes_xbk_inc.md -> archive/recipes_xbk_inc.md
	renamed:    docs/archive/ssh_server.md -> archive/ssh_server.md
	modified:   docs/create-compressed-backup.md
	modified:   docs/take-streaming-backup.md
	modified:   docs/xbstream-binary-overview.md
	modified:   docs/xtrabackup-option-reference.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
LGTM

File: docs/create-compressed-backup.md [hunk 1]
```diff
@@ -1,79 +1,136 @@
...
 
-!!! note
+To make a compressed backup, use the `--compress` option along with the `--backup` and `--target-dir` options. 
 
-    Starting with Percona XtraBackup 8.0.31-24 using qpress/QuickLZ to compress backups is deprecated and may be removed in future versions. We recommend using either `LZ4` or Zstandard (`ZSTD`) compression algorithms.
+## Version updates
 
-To make a compressed backup, use the `--compress` option along
-with the `--backup` and `--target-dir` options. 
+??? note "Starting with Percona XtraBackup 8.0.34-29"
```
Comment (altmannmarcelo) [inline docs/create-compressed-backup.md:9]: What does `???` means here?

File: docs/create-compressed-backup.md [hunk 2]
```diff
@@ -1,79 +1,136 @@
...
+    ```
    
-    Enable the repository: `percona-release enable-only tools release`.
+    You can specify `ZSTD` compression level with the [`--compress-zstd-level(=#)`](xtrabackup-option-reference.md#compress-zstd-level) option. The default value is `1`.
 
-If *Percona XtraBackup* is intended to be used in combination with
-the upstream MySQL Server, you only need to enable the `tools`
-repository: `percona-release enable-only tools`.
+    ```{.bash data-prompt="$"}
+    $ xtrabackup --backup --compress-zstd-level=1 --target-dir=/data/backup
```
Comment (altmannmarcelo) [inline docs/create-compressed-backup.md]:
```suggestion
    $ xtrabackup --backup --compress --compress-zstd-level=1 --target-dir=/data/backup
```

File: docs/create-compressed-backup.md [hunk 3]
```diff
@@ -1,79 +1,136 @@
...
-```
+    ```{.bash data-prompt="$"}
+    $ xtrabackup --backup --compress=zstd --target-dir=/data/backup
+    ```
+   
+    You can specify `ZSTD` compression level with the [`--compress-zstd-level(=#)`](xtrabackup-option-reference.md#compress-zstd-level) option. The default value is `1`.
 
-??? example "Expected output"
+    ```{.bash data-prompt="$"}
+    $ xtrabackup --backup --compress-zstd-level=1 --target-dir=/data/backup
```
Comment (altmannmarcelo) [inline docs/create-compressed-backup.md]:
```suggestion
    $ xtrabackup --backup --compress --compress-zstd-level=1 --target-dir=/data/backup
```

File: docs/take-streaming-backup.md [hunk 4]
```diff
@@ -20,38 +20,7 @@ $ xtrabackup --stream=xbstream --target-dir=/tmp
...
-[Percona XtraBackup 8.0.30-23](release-notes/8.0/8.0.30-23.0.md) adds support for the `Zstandard (ZSTD)` compression algorithm. `ZSTD` is a fast lossless compression algorithm that targets real-time compression scenarios and better compression ratios. To compress files using the `ZSTD` compression algorithm, use the `--compress=zstd` option. The resulting files have the `\*.zst` format. 
-    
-You can specify `ZSTD` compression level with the [`--compress-zstd-level(=#)`](xtrabackup-option-reference.md#compress-zstd-level) option. The defaul value is `1`.
-
-```{.bash data-prompt="$"}
-$ xtrabackup --backup --compress-zstd-level=1 --target-dir=/data/backup
-```
-    
-To decompress files, use the `--decompress` option.
+When compression is enabled, *xtrabackup* compresses the output data, except for the meta and non-InnoDB files which are not compressed, using the specified compression algorithm. Read about the supported compression algorithms in [Create a compressed backup](create-compressed-backup.md) document.
```
Comment (altmannmarcelo) [inline docs/take-streaming-backup.md]:
```suggestion
When compression is enabled, *xtrabackup* compresses the output data, except for the metadata, using the specified compression algorithm. Read about the supported compression algorithms in [Create a compressed backup](create-compressed-backup.md) document.
```

Comment (altmannmarcelo) [reply docs/take-streaming-backup.md]: Non-InnoDB tables are also compressed:

```bash
 🐬 marcelo  📂 ~  ▶
 ╰▶ $ ls /tmp/backup/test/
tb1.MYD.zst  tb1.MYI.zst  tb1_377.sdi.zst
```

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

File: docs/create-compressed-backup.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/create-compressed-backup.md:9]: This is the markdown syntax we use to add tabs.

File: docs/take-streaming-backup.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/take-streaming-backup.md]: Updated

File: docs/create-compressed-backup.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/create-compressed-backup.md]: fixed

File: docs/create-compressed-backup.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/create-compressed-backup.md]: fixed

**Pull request merged**